packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -16,6 +16,20 @@ export interface TypingIndicatorStylesProps extends BaseCustomStylesProps {
   typingString?: IStyle;
 }
 
+/**
+ * Strings of TypingIndicator that can be overridden
+ */
+export interface TypingIndicatorStrings {
+  /** String to use when one user is typing */
+  singular: string;
```

Comment:
It's not clear from the type definition how these strings will be used.

Could you provide an example string for each (in english) along with users that are typing, and show what the result will be. i.e., do we format the users with the string, concat it... ?

EDIT: I see from the tests that these strings have formatting placeholders. This is great. Please document those placeholders, preferably with example.